### PR TITLE
NOTICKET tidy

### PR DESCRIPTION
**Pull request by FostUK**
* Fix dependency vulnerabilities.
* Remove webpack.
* Remove Python serve. 
* Remove "Senior" from title as used for range of roles.
* Misc typo fixes

### CODE REVIEW

**Comment by AlyssaXA** [comment]
There's a webpack config file you can remove too

**Comment by AlyssaXA** [comment]
This removes the build too, no `dist` files will be produced either

**Comment by FostUK** [comment]
> This removes the build too, no `dist` files will be produced either

Good point! Have removed the dist folder and webpack config. Can you test on Mac when you have a min @AdamXA ?

File: README.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 ## Building and running locally
 
-```javascript
+```shell
 $ git clone git@github.com:bbc/ce-tech-test.git && cd ce-tech-test
 $ npm install
 $ npm run build
-$ npm run serve # or "npm run serve27" if this errors
+$ npm run serve
 ```
```
Comment (AlyssaXA) [inline README.md]: `npm run build` no longer exists

**Comment by AlyssaXA** [comment]
> > This removes the build too, no `dist` files will be produced either
> 
> Good point! Have removed the dist folder and webpack config. Can you test on Mac when you have a min @AdamXA ?

Not working for me, getting 404 on `http://localhost:8000/dist/bundle.js`

File: README.md (hunk 1 above)
Comment (FostUK) [reply README.md]: Ta.

**Comment by FostUK** [comment]
> > > This removes the build too, no `dist` files will be produced either
> > 
> > 
> > Good point! Have removed the dist folder and webpack config. Can you test on Mac when you have a min @AdamXA ?
> 
> Not working for me, getting 404 on `http://localhost:8000/dist/bundle.js`

Ah. Needs the direct import adding now the build has gone.

**Comment by FostUK** [comment]
Updated to load files directly. Can you double check on Mac @AdamXA . Thanks

**Comment by AlyssaXA** [comment]
Seems all good now

**Review by astalker — APPROVED** [review]
Thanks!

**Pull request merged**